### DISCUSSION TITLE
Financial Support Options?

### DISCUSSION THREAD

**Discussion by tjluoma** [swiftbar/SwiftBar#92] [General]
Let me start off by saying that I am thrilled that this project exists and hope that it continues to have a long and fruitful life.

Then let me add: And I really don’t want it to die off or dry up because it had no financial support. I hope that year from now this project isn’t 9 months idle because the initial burst of steam faded out as it became more of a chore than a joy. Looking back over the history of BitBar, it seems there was a huge flurry of activity at the beginning, and then it sort of died off. 

I don't really know anything about software development, but ISTM that it wouldn't be a terrible idea to have an option for people who want to financially support the project to do so. Maybe as it grows, so will its support base.

And what do they get in return for this? In the immortal words of John Siracusa: ***nothing***. I'm not suggesting that this would keep the app from being freely distributed, or that donors would get any sort of perks or whatever. I'm just saying that if you gave me a way to throw you some dollars, I would, and I hope others would too (especially those of us who have neither the skills nor the time to contribute code, for example).

Just a thought!

**Comment by rockyzhang24** [comment]
I second the financial support and some way like donate.

**Comment by melonamin** [comment]
Hey guys,

GitHub Sponsors application is pending for SwiftBar organization at the moment, thank you for your support

**Comment by xilopaint** [comment]
I can't say how happy I am to know this project and would like to make a donation as soon as GitHub Sponsors feature is set up. The milestone feature which tracks the progress of development is AMAZING.

**Reply by rockyzhang24** [reply]
Same here! Cannot wait to make a donation to support this project and the author's efforts.

**Comment by rockyzhang24** [comment]
@melonamin Hi, 

I really want to sponsor this amazing repo but I cannot find a method to do it. When will the GitHub Sponsors feature be set up? Cannot wait! 🤗
